### PR TITLE
Feature: 59 - added details about definition of ready

### PR DESCRIPTION
**Pull request by madossan01**
Added topics covering:
•	What it is
•	Who writes it
•	When a Definition of Ready needs an update
•	What should be avoided

### CODE REVIEW

File: team-agreements/definition-of-ready/readme.md [hunk 1]
```diff
@@ -1,8 +1,30 @@
...
+
+## What it is
+
+Definition of Ready is the agreement made by the scrum team around how complete a user story should be to be selected as candidate for estimation in the sprint planning.
+
+It can be understood as a checklist that helps the Product Owner to ensure that the user story he wrote contains all the necessary and the enough content to the scrum team.
+
+### Examples of ready checklist items:
+
+- The description had enough details and input values mandatory in the implementation.
```
Comment (TessFerrandez) [inline team-agreements/definition-of-ready/readme.md]: Suggestion for the checklist items (for consistency with other checklist items in the playbook)

- [  ] Does the description have the details and input values required to implement the story?
- [  ] Does the user story have clear and complete acceptance criteria?
- [  ] Does the user story address the business need?
- [  ] Can we measure the acceptance criteria?

File: team-agreements/definition-of-ready/readme.md [hunk 2]
```diff
@@ -1,8 +1,30 @@
...
+- The user story contained a clear and a complete acceptance criteria.
+- The user story addressed the business view
+- The acceptance criteria could be measured
+
+## Who writes it
+
+The ready checklist can be written by a Product Owner in agreement with the development team and the scrum master.
+
+## When a Definition of Ready needs an update
```
Comment (TessFerrandez) [inline team-agreements/definition-of-ready/readme.md]: When a Definition of Ready needs an update -> When should the Definition of Ready be updated

File: team-agreements/definition-of-ready/readme.md [hunk 3]
```diff
@@ -1,8 +1,30 @@
...
+
+You can update or change the definition of ready anytime the scrum team observes that there are missing information in the user stories that recurrently impact the good evolution of the planning and that recurrent information was not in the checklist.
```
Comment (TessFerrandez) [inline team-agreements/definition-of-ready/readme.md]: that recurrently impact the good evolution of the planning and that recurrent information was not in the checklist -> that impacts the planning

Comment (TessFerrandez) [inline team-agreements/definition-of-ready/readme.md]: Nit: You can update -> Update

File: team-agreements/definition-of-ready/readme.md [hunk 4]
```diff
@@ -1,8 +1,30 @@
...
+
+## What should be avoided
+
+It is desirable that the ready checklist can apply to two or more user stories. We discourage you of including in your ready checklist any detail that make the ready checklist to become one of a kind and too specific for a single user story. This may lead to a 1:1 relationship between the user story and the checklist, what may delay the process of writing good user stories.
```
Comment (TessFerrandez) [inline team-agreements/definition-of-ready/readme.md]: Suggestion for reword:

The ready checklist should contain items that apply broadly.  Don't include items that only apply to one or two user stories.

File: team-agreements/definition-of-ready/readme.md [hunk 5]
```diff
@@ -1,8 +1,30 @@
...
 
-## Content TODO's
+At the time the development team picks a user story from the top of the backlog to estimate during the Sprint planning, they consider that the user story has a certain level of detail that helps them to give an accurate estimate of completion of the user story within the sprint. This level of information is what makes the story to be Ready to be developed in the sprint.
 
-- Define criteria for a story to be ready to pull into a sprint.
-  - Examples
-    - Has estimate
-    - Team understands implementation/design
+  `In case a user story is not Ready in the beginning of the Sprint it increases the chance that your story will not be done at the end of this print.`
+
```
Comment (TessFerrandez) [inline team-agreements/definition-of-ready/readme.md]: Suggestion for reword to make it a bit more concise:

When the development team picks a user story from the backlog, the user story needs to have enough detail to estimate the work needed to complete the story. If it has enough detail to estimate, it is Ready to be developed in the sprint.

If the user story is not Ready in the beginning of the Sprint, there is a bigger chance that the user story will not be Done by the end of the sprint.

**Comment by fnocera** [comment]
@madossan01 please could you add a label and project backlog linking (on the right hand side) when you open a PR please. Also please make sure you link to the issue that your PR is addressing. Thanks :)

File: team-agreements/definition-of-ready/readme.md (hunk 1 above)
Comment (madossan01) [reply team-agreements/definition-of-ready/readme.md]: done. Thanks, @TessFerrandez

File: team-agreements/definition-of-ready/readme.md (hunk 2 above)
Comment (madossan01) [reply team-agreements/definition-of-ready/readme.md]: Done. Thanks.

File: team-agreements/definition-of-ready/readme.md (hunk 3 above)
Comment (madossan01) [reply team-agreements/definition-of-ready/readme.md]: Done. Thanks

Comment (madossan01) [reply team-agreements/definition-of-ready/readme.md]: Done. Thanks,

File: team-agreements/definition-of-ready/readme.md (hunk 4 above)
Comment (madossan01) [reply team-agreements/definition-of-ready/readme.md]: Done. Thanks.

File: team-agreements/definition-of-ready/readme.md (hunk 5 above)
Comment (madossan01) [reply team-agreements/definition-of-ready/readme.md]: Done. Thanks.

**Comment by bsherwin** [comment]
@madossan01 I made some changes to some sentence structure. If you are good with them, you can complete the Pull Request.

**Pull request merged**